daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/WarningLog.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.speedy
+
+import org.slf4j.Logger
+import scala.collection.mutable.ArrayBuffer
+
+private[lf] final class WarningLog(logger: Logger) {
+  private[this] val buffer = new ArrayBuffer[String](initialSize = 10)
+  def add(message: String): Unit = {
+    if (logger.isWarnEnabled) {
```

Comment:
I don’t think you need this. It makes sense for debug logs where you construct potentially expensive messages but here the string is already materialized and we expect warnings to always be on.

Reply:
I'll remove it, thanks!